packages/ses/test/tame-global-error-object.test.js

```diff
@@ -5,17 +5,18 @@ import tameGlobalErrorObject from '../src/tame-global-error-object.js';
 const { test } = tap;
 
 test('tameGlobalErrorObject', t => {
-  t.plan(3);
-
   const restore = captureGlobals('Error');
 
-  tameGlobalErrorObject();
-
-  t.equal(Error.captureStackTrace, undefined);
-  t.equal(Error.stackTraceLimit, undefined);
-
-  const error = new Error();
-  t.equal(error.stack, undefined);
+  try {
+    tameGlobalErrorObject();
 
-  restore();
+    t.equal(typeof Error.stackTraceLimit, 'number');
```

Comment:
do we want a test in here that `Error.stackTraceLimit = xx` doesn't throw, to exercise what `tap` does as it tries to display an async error?